contrib/jenkins_tests/build.sh

```
@@ -13,15 +13,22 @@ cd ${build_dir}
 declare -A build_list
 build_list['debug']="--enable-opt-log=no --enable-debug"
 build_list['nginx-off']="--enable-nginx=no"
 build_list['envoy-on']="--enable-nginx=yes"
 build_list['static-on']="--enable-static --disable-shared"
+build_list['werror']="CFLAGS=\"${werror_cflags}\""
 build_list['default']=""
 
 build_tap=${WORKSPACE}/${prefix}/build.tap
-echo "1..${#build_list[@]}" > $build_tap
+# +1 for the exported-CFLAGS build below
+echo "1..$((${#build_list[@]} + 1))" > $build_tap
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)